docs/book/src/topics/secondary-load-balancer.md

```diff
@@ -34,3 +34,30 @@ spec:
     name: internal-apiserver
     scheme: internal     # optional
 ```
+
+## Target Group IP Address Type
+
+The secondary load balancer supports the same `targetGroupIPType` configuration as the primary load balancer. By default, the target group IP address type is set based on the VPC configuration:
+- If the VPC has IPv6 enabled, the target group uses `ipv6`
```

Comment:
TODO review: if we look at VPC level, dual-stack would always have IPv6 enabled, even in dual-stack mode with primary ipv4?